crates/ere-hosts/src/main.rs

```diff
@@ -182,6 +117,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         force_rerun: cli.force_rerun,
     };
 
+    println!("Running benchmarks with resource: {:?}", resource);
```

Comment:
Not something to change in this PR (maybe create an issue): We can switch println to trace, so that we can silence certain outputs and mark certain outputs as "info/warning/error" etc